src/backend/events/app-removal/event.ts

```diff
@@ -0,0 +1,19 @@
+import { appInstances } from '@wix/app-management';
+import { auth } from '@wix/essentials';
+import { getMiddlewareUrl } from '../../../utils/api';
+
+appInstances.onAppInstanceRemoved(async () => {
```

Comment:
This is an initial test, not working quite yet, will pick it up in a separate PR.